orc8r/tools/ansible/roles/pkgrepo/tasks/main.yml

```diff
@@ -14,6 +14,10 @@
   fail: msg="distribution is undefined for the pkgrepo role"
   when: distribution is undefined
 
+- name: remove fluentbit repo as its broken
+  shell: rm -rf /etc/apt/sources.list.d/packages_fluentbit_io_ubuntu_focal.list
+  become: yes
```

Comment:
can you add ignore_error, when we will rebuild the VM, it will fail

Reply:
yup